tenant/storage_bucket.go

```diff
@@ -146,16 +146,10 @@ func (s *Store) ListBuckets(ctx context.Context, tx kv.Tx, filter BucketFilter,
 		return nil, invalidBucketListRequest
 	}
 
-	// if we dont have any options it would be irresponsible to just give back all orgs in the system
 	if len(opt) == 0 {
-		opt = append(opt, influxdb.FindOptions{
-			Limit: influxdb.DefaultPageSize,
-		})
+		opt = append(opt, influxdb.FindOptions{})
```

Comment:
This feels like something that should have had a test - can we add a test for > 20 buckets returned?